libs/application/template-api-modules/src/lib/modules/templates/institution-application/emailGenerators/applicationEmail.ts

```diff
@@ -0,0 +1,63 @@
+import { dedent } from 'ts-dedent'
+import get from 'lodash/get'
+import { EmailTemplateGeneratorProps } from '../../../../types'
+import { applicationOverviewTemplate } from './applicationOverviewTemplate'
+import { SendMailOptions } from 'nodemailer'
+
+export interface NodemailAttachment {
+  filename: string
+  href: string
+}
+
+interface ApplicationEmail {
+  (
+    props: EmailTemplateGeneratorProps,
+    applicationSender: string,
+    applicationRecipient: string,
+  ): SendMailOptions
+}
+
+export const generateApplicationEmail: ApplicationEmail = (
+  props,
+  applicationSender,
+  applicationRecipient,
+): SendMailOptions => {
+  const {
+    application,
+    options: { locale },
+  } = props
+
+  const institutionName = get(application.answers, 'applicant.institution')
+
+  const subject = `Umsókn frá ${institutionName}`
+  const attachments = get(application.answers, 'attatchments') as []
+
+  const mailAttachments = attachments
+    ? attachments.map(
+        (attachment) =>
+          ({
+            filename: get(attachment, 'name'),
+            href: `${get(attachment, 'url')}/${get(attachment, 'key')}`,
```

Comment:
I would try using typescript optional chaining rather than lodash/get when possible. eg
```typescript
const URL = attachment?.url; // string | undefined
```